examples/tutorials/core_api_pytorch_mnist/model_def_metrics.py

```diff
@@ -94,14 +112,20 @@ def test(args, model, device, test_loader, epoch, core_context, steps_completed)
 
     print(
         "\nTest set: Average loss: {:.4f}, Accuracy: {}/{} ({:.0f}%)\n".format(
-            test_loss, correct, len(test_loader.dataset), 100.0 * correct / len(test_loader.dataset)
+            test_loss,
+            correct,
+            len(test_loader.dataset),
+            100.0 * correct / len(test_loader.dataset)
+            # Docs snippet end before: include args
```

Comment:
There are two closing parens not included in this snippet.

Reply:
fixed